File: RGB to Hex/RGB to Hex.py
def rgb_to_hex(rgb_str):
    # Remove "rgb(" and ")" then split into integers
    r, g, b = map(int, rgb_str.strip("rgb()").split(","))
    # Format each value as 2-digit hex and concatenate
    return "#{:02x}{:02x}{:02x}".format(r, g, b)

File: RGB to Hex.dart
String rgbToHex(String rgbStr) {
  // remove "rgb(" and ")" then split into numbers
  final values = rgbStr
      .replaceAll(RegExp(r'rgb|\(|\)|\s'), '')
      .split(',')
      .map(int.parse)
      .toList();

  final r = values[0];
  final g = values[1];
  final b = values[2];

  // format each value as 2-digit hex
  return '#${r.toRadixString(16).padLeft(2, '0')}' +
         '${g.toRadixString(16).padLeft(2, '0')}' +
         '${b.toRadixString(16).padLeft(2, '0')}';
}



